dashboard/server/methods.js

```diff
@@ -1,18 +1,84 @@
-import { Meteor } from 'meteor/meteor';
-import ES from 'elasticsearch';
-import config from '/config';
+import { Meteor } from 'meteor/meteor'
+import ES from 'elasticsearch'
+import _ from 'lodash'
+import moment from 'moment'
+import config from '../../config'
+
+const client = new ES.Client({
+  host: config.host
+})
+
+const types = [
+  'on_client_connected',
+  'on_client_disconnected',
+  'on_client_subscribe',
+  'on_client_unsubscribe',
+  'on_session_created',
+  'on_session_subscribed',
+  'on_session_unsubscribed',
+  'on_session_terminated',
+  'on_message_publish',
+  'on_message_delivered',
+  'on_message_acked'
+]
 
 Meteor.methods({
-  getEsData (opts) {
+  getAggr (opts) {
```

Comment:
While this function name pretty much describes what it does, it should also be [meaningful and pronouncable](https://github.com/ryanmcdermott/clean-code-javascript#use-meaningful-and-pronounceable-variable-names). `getAggr` is *almost* meaningful and *sort-of* pronouncable, but would be much better as `getAggregateData`.